Orlayhemmy/EventManager
# API documentation

### What does this PR do
It adds the documentation for the API used in the application

### Description of tasks to be completed
Add documentation using swagger
Remove unnecessary files
Replace test snapshots

### How should it be tested
Go to this [https://ecenter.eventmanager.heroukapp.com/api-docs](link ) and check it out. Click on each of the API endpoints and try it out by filling in the necessary details.

### Screenshots
<img width="1440" alt="screen shot 2018-07-13 at 9 56 02 am" src="https://user-images.githubusercontent.com/33638254/42682614-063a7c5e-8683-11e8-9a59-e38715067267.png">



### What are the relevant pivotal tracker stories
#159030275
